test/extension-tests/successful-build/test/variable-substitution.test.ts

```diff
@@ -138,6 +139,12 @@ suite('Variable Substitution', () => {
                 '[cmakeInstallPrefix] substitution incorrect');
         expect(typeof cacheEntry.value).to.eq('string', '[cmakeInstallPrefix] unexpected cache entry value type');
 
+        // Check substitution for "pathListSep".
+        cacheEntry = cache.get('pathListSep') as CacheEntry;
```

Comment:
I think you should revert the changes to this test file.  pathListSep only exists in the context of presets, it's not something you can use for substitution in the settings.